keras_contrib/applications/densenet.py

```diff
@@ -406,7 +415,7 @@ def __transition_up_block(ip, nb_filters, type='upsampling', output_shape=None,
     elif type == 'subpixel':
         x = Conv2D(nb_filters, (3, 3), padding="same", kernel_regularizer=l2(weight_decay), activation='relu',
                    use_bias=False, kernel_initializer='he_uniform')(ip)
-        #x = Convolution2D(nb_filters, 3, 3, activation="relu", border_mode='same', W_regularizer=l2(weight_decay),
```

Comment:
I'm keeping Keras 1 code for now, since it is cross compatible with Keras 2. Not the other way around.

I need to use Keras 1.2.2 for the time being, since I need many features which were abruptly removed for random reasons from Keras 2 (such as Batch norm mode 2 which I need for GANs, fbeta_score metrics, and a few bugs which have come up in 2 are making me wait for the shift). 

I will update the code when Keras 2 stabilizes (bug free), and fixes some of its mistakes like Mode 2 batchnorm. Until then, better to keep code compatible with both versions.